packages/core/src/api/blockManipulation/selections/selectioncut.test.ts

```diff
@@ -0,0 +1,314 @@
+import { TextSelection } from "prosemirror-state";
+import { afterAll, beforeAll, describe, expect, it } from "vitest";
+import { PartialBlock } from "../../../blocks/defaultBlocks.js";
+import { BlockNoteEditor } from "../../../editor/BlockNoteEditor.js";
+
+// describe("Test getSelectionCutBlocks", () => {
+  const initialContent: PartialBlock[] = [
```

Comment:
@matthewlipski could you make these tests in line with your new testing setup?

I think they were originally copied from existing tests - so you can probably reuse existing test document / selection data